en/patterns/form.md

```diff
@@ -6,15 +6,15 @@ _keywords: Design System, Sketch, Ignite UI for Angular, Pattern, UI Library, Wi
 
 ## Form
 
-Use the Form Pattern to create meaningful application scenarios, where the collection of user input is necessary. The various data entry and display Components are used to constitute some of the most common and meaningful Forms.
+The Form Pattern comes with the styling flexibility provided by the Input, Checkbox, Button, Hyperlink, and Text that constitute its layout.
```

Comment:
@StefanIvanov Doesn't this change make this line equivalent to line 13?  Is that intentional?